frontend/src/App.tsx

```diff
@@ -1,16 +1,24 @@
 import { FC, ReactElement } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
It is fine to have <BrowserRouter> in the code - please do not rename it to "Router" as it may be confusing and not easy to read